src/test/feature/testhawqregister/test-hawq-register.cpp

```diff
@@ -0,0 +1,80 @@
+#include <string>
+
+#include "lib/command.h"
+#include "lib/common.h"
+#include "lib/sql-util.h"
+
+#include "gtest/gtest.h"
+
+using std::string;
+
+/* This test suite may consume more than 80 seconds. */
+class TestHawqRegister : public ::testing::Test {
+ public:
+  TestHawqRegister() {}
+  ~TestHawqRegister() {}
+};
+
+TEST_F(TestHawqRegister, TestSingleFile) {
+	SQLUtility util;
+	string rootPath(util.getTestRootPath()); 
+	string relativePath("/testhawqregister/test_hawq.paq");
+	string filePath = rootPath + relativePath;
+
+	EXPECT_EQ(Command::getCommandStatus("hadoop fs -put " + filePath + " /hawq_register_hawq.paq"), 0);
```

Comment:
EXPECT_EQ(expected_value, your_value), so zero should be the first parameter.
some other EXPECT_EQ has the same problem.